zips/zip-0400.rst

```diff
@@ -43,46 +43,61 @@ Here ``zkey`` is the property name located at the first position of the database
 Schema
 ------
 
-According to Zcash v3.0.0-rc1 the following key-values can be found: the property names in **bold** mean only one instance of this type can exist in the entire database, while the others, suffixed by '\*' can have multiple instances. Keys and Values columns of the table contain the types that the stored data is representing. Included also there are the variable names hoping it will add some clarity to what the stored data is representing.
+The following key-values can be found: the property names in **bold** mean only one instance of this type can 
+exist in the entire database, while the others, suffixed by '\*' can have multiple instances. Keys and Values columns of the table
+contain the types that the stored data is representing. The Version column indicates the zcashd version the key is present. 
```

Comment:
```suggestion
contain the types that the stored data is representing. The Version column indicates the zcashd version(s) that create databases containing this key.
```
The zcashd versions that create a key could potentially be different from the versions that understand it.